docs/src/ref_impl/build_and_run.md

```diff
@@ -161,3 +162,25 @@ In the case of i.MX8, Ghaf deployment contains of creating a bootable SD card wi
    2. Prepare the USB boot media with the target HW image you built: `dd if=./result/nixos.img of=/dev/<YOUR_USB_DRIVE> bs=32M`.
    
 3. Insert an SD card and USB boot media into the board and switch the power on.
+
+---
+
+### Building Ghaf Image for Microchip Icicle kit
+
+Before you begin, check device-independent [prerequisites](./build_and_run.md#prerequisites).
+
+In the case of Icicle kit, Ghaf deployment contains of creating a SD Image which includes UBoot and Linux kernel from Microchip and Ghaf based NixOS rootfs:
+
+1. To build Ghaf SD image:
+   Run the `nix build .#packages.riscv64-linux.mpfs-icicle-kit-release` command.
```

Comment:
Should we also instruct here to use the `-debug` target? Considering other reference targets, `-release` is of very limited usefulness. Also, all the possible debug channels (ssh|serial|..) should go to `-debug`-targets.

Reply:
Added instructions to build image for debug target too.